.env.test

```diff
@@ -35,3 +35,4 @@ STAGING_OAUTH_SECRET=example-secret
 AVATARS_PUBLIC_STORAGE=local
 AVATARS_PRIVATE_STORAGE=local_private
 DUMP_HOST=https://assets.worldcubeassociation.org
+CURRENCY_LAYER_API_KEY=9d254ec37e1591eec964dac947fafa09
```

Comment:
Are these keys (here in `test` and same in `dev`) random dummy values or are they actual access keys?

Reply:
We actually use access keys from testing or dummy accounts for Stripe and PayPal. Look into `.env.test`, you will see that a lot of `PAYPAL_*` variables exist. These are from "real" accounts, which means we have actual username/password to log in to PayPal dashboard. But if you do actually log in, you will see that these accounts are marked as test accounts by PayPal, and you cannot send actual, real money with them. Only sandbox money. So that's why the accounts are not real, they are only "real".

Reply:
Okay, I think currencylayer doesn't have a test account as it doesn't do any sensitive things. So will it make sense to create two currencylayer accounts for WCA - one for test and one for production?

Reply:
Hm, now that I think about it, I have another idea...
What would you think about using the `if Rails.env.test?` check to use the `EuCentralBank` in local environments and switch on the CurrencyLayer in production only?

Reply:
You can introduce a flag in `EnvConfig` (file `env_config.rb`) where developers can switch on CurrencyLayer manually if they want, for local testing and stuff

Reply:
That is a great idea, I've now implemented it. :-)